### ISSUE TITLE
Stop working in Sketch 3.5.1

### ISSUE THREAD

**Issue by hudochenkov** [tylergaw/day-player#31]
Hi!

There is apparently breaking changes in new Sketch update.

**Comment by LukePettway** [comment]
:(

**Comment by jarace87** [comment]
Looks like MSImageProxy has gone:

`Sketch: Exceptioin: ReferenceError: Can't find variable: MSImageProxy`

https://github.com/BohemianCoding/developer.sketchapp.com/issues/32

**Comment by jensbjork** [comment]
+1

**Comment by LukePettway** [comment]
Just out of curiosity, are you using Sketch Toolbox to manage the plugin or manually adding the files?

**Comment by aatrostle** [comment]
Running into this issue on Sketch Version 3.5.2 (25235). First tried installing via Sketch Toolbox, plugin wasn't working. Uninstalled via Sketch Toolbox then installed via git to `~/Library/Application Support/com.bohemiancoding.sketch3/Plugins`, plugin still wasn't working :disappointed: 

I am unsure how to dump logs or whatever might be helpful for debug.

**Comment by LukePettway** [comment]
@aatrostle Good to know, I've had issues with Sketch Toolbox, it looks like it isn't working anymore and doesn't auto-update at least since the last time I checked a week or so ago. 

Can you run console.app and see if they are logging any output to that?

**Comment by aatrostle** [comment]
Ahh, Console.app, good to know what thats for haha.

```
2/18/16 7:18:27.728 PM Placehold.it (Sketch Plugin)[60350]: ReferenceError: Can't find variable: MSImageProxy
```

Seems to be the same issue mentioned earlier in this thread..

**Comment by LukePettway** [comment]
Well in the mean time I recommend checking out Craft:

http://www.invisionapp.com/craft

**Comment by geligelu** [comment]
Anyone reviewed @hjue's commit?

**Comment by tylergaw** [comment]
Thanks ya'll. Sorry for the slowness getting back to this. Had neglected the project for some time. I just released 3.0.0 which updates the plugin for the new plugin setup/api in Sketch 40+. Closing this out now.
